jax/api.py

```diff
@@ -868,6 +873,10 @@ def jacfwd(fun: Callable, argnums: Union[int, Sequence[int]] = 0,
    [ 1.6209   0.       0.84147]]
   """
   _check_callable(fun)
+  if np.isscalar(argnums):
```

Comment:
Rather than repeating this code, put it in a helper? I'd rather not duplicate parsing code.

Reply:
As above, the code furthur down branches on whether the input is a tuple or a scalar, so we can't convert it to a tuple here.,